keyformproject/settings.py

```diff
@@ -41,6 +41,7 @@
     'django.contrib.staticfiles',
     'keyform',
     'debug_toolbar',
+    'mathfilters'
```

Comment:
If we are going to use this does it needed to be added to the requirements folder?